### ISSUE TITLE
Loss of attributes without values

### ISSUE THREAD

**Issue by exeto** [posthtml/posthtml-render#1]
input:
```js
{
  tag: 'input',
  attrs: {
    type: 'checkbox',
    checked: ''
  }
}
```
output:
```html
<input type="checkbox">
```

**Comment by voischev** [comment]
@exeto this a fueatute ;) `'' == false` https://github.com/posthtml/posthtml-render/blob/master/lib/posthtml-render.js#L80

use boolean value
```
{
  tag: 'input',
  attrs: {
    type: 'checkbox',
    checked: true
  }
}
```

**Comment by voischev** [comment]
@exeto fix in [v1.0.3](https://github.com/posthtml/posthtml-render/commit/7a74dd692d1b2aa4a209dcc684dfa30258042da0)

**Comment by exeto** [comment]
Thanks
